File: packages/action-sheet/ios/Plugin/Classes/Results/ShowActionsResult.swift
import Foundation
import Capacitor

@objc public class ShowActionsResult: NSObject, Result {
    let index: Int

    init(index: Int) {
        self.index = index
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["index"] = index
        return result as AnyObject
    }
}

File: packages/action-sheet/android/src/main/java/io/capawesome/capacitorjs/plugins/actionsheet/classes/results/ShowActionsResult.java
package io.capawesome.capacitorjs.plugins.actionsheet.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.actionsheet.interfaces.Result;

public class ShowActionsResult implements Result {

    private final int index;

    public ShowActionsResult(int index) {
        this.index = index;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("index", index);
        return result;
    }
}
